### PR TITLE
Remove netchannel as it has been released + mirage-solo5 new dependency

### PR DESCRIPTION
**Pull request by TheLortex**

### CODE REVIEW

**Comment by TheLortex** [comment]
Rebased + updated mirage-solo5 dependency

File: packages/mirage-solo5/mirage-solo5.0.5.0+dune/opam [hunk 1]
```diff
@@ -23,6 +23,7 @@ depends: [
   "lwt" {>= "2.4.3"}
   "ocaml-freestanding" {>= "0.4.4"}
   "mirage-os-shim"
+  "metrics"
```
Comment (Julow) [inline packages/mirage-solo5/mirage-solo5.0.5.0+dune/opam]: This is unrelated. Can you submit 2 PRs?

Comment (TheLortex) [reply packages/mirage-solo5/mirage-solo5.0.5.0+dune/opam]: Tool late
